.gitlab-ci.yml

```diff
@@ -708,4 +711,4 @@ trigger-simnet:
     DWNSTRM_ID:                    332
   script:
     # API trigger for a simnet job
-    - .maintain/gitlab/trigger_pipeline.sh
+    - .maintain/gitlab/trigger_pipeline.sh --simnet-version=${SIMNET_REF}
```

Comment:
Where is this `SIMNET_REF` defined?

Reply:
This is defined as a variable in gitlab https://gitlab.parity.io/parity/substrate/-/settings/ci_cd